### DISCUSSION TITLE
Is there events on cursor move up/down action?

### DISCUSSION THREAD

**Discussion by linrongbin16** [junegunn/fzf#3420] [Q&A] [answered]
Hi, I checked the manual here: https://man.archlinux.org/man/fzf.1.en, there are many events:
* start
* load
* focus
* one
* ...

When press up/down (ctrl-n/ctrl-p) to move up/down, will it trigger the `focus` event?

If no, is there events on cursor move up/down action?

**Comment by LangLangBart** [comment, marked as answer]
- Have you encountered any problems? Your question is formulated in such a way that the answer is already obvious by executing a simple demo command. 

```sh
fzf --bind 'ctrl-p:up,ctrl-n:down' --bind 'focus:change-preview:echo yes'
```

**Reply by linrongbin16** [reply]
oh, thanks, it should be `focus` changed
